### ISSUE TITLE
[Enhancement][Grid] NVDA does not read table name

### ISSUE THREAD

**Issue by filipKovachev** [telerik/kendo-react#1862]
## I'm submitting a...

* Suggestion for improvement

## Current behavior
Currently, NVDA reads 'table with 78 rows and 5 columns  row 1  column 1  ID' instead of 'Brazil table  with 78 rows and 5 columns  row 1  column 1  ID'.

## Expected behavior
NVDA should narrate 'Brazil table with 78 rows and 5 columns  row 1  column 1  ID'.

## Minimal reproduction of the problem with instructions
https://stackblitz.com/edit/react-7zidww-yw7ard?file=app%2Fmain.tsx

Ticket ID: 1633735

**Comment by iadnonov** [comment]
Our accessibility team exploring the most effective approach.

**Comment by iadnonov** [comment]
Fixed in dev version [v7.2.4-develop.1](https://www.telerik.com/kendo-react-ui/components/changelogs/ui-for-react/#v7.2.4-develop.1)
